scene/gui/split_container.cpp

```diff
@@ -309,24 +304,34 @@ void SplitContainer::_notification(int p_what) {
 		case NOTIFICATION_LAYOUT_DIRECTION_CHANGED: {
 			queue_sort();
 		} break;
-
 		case NOTIFICATION_SORT_CHILDREN: {
 			_resort();
 		} break;
-
 		case NOTIFICATION_THEME_CHANGED: {
 			update_minimum_size();
 		} break;
+		case NOTIFICATION_DRAW: {
+			draw_style_box(theme_cache.split_bar_background, split_bar_rect);
+			if (show_drag_area && Engine::get_singleton()->is_editor_hint()) {
+				draw_rect(dragging_area_control->get_rect(), dragging_enabled ? Color(1, 1, 0, 0.3) : Color(1, 0, 0, 0.3));
+			}
+			if (dragger_visibility != DRAGGER_VISIBLE || (!dragging_area_control->dragging && !dragging_area_control->mouse_inside && theme_cache.autohide)) {
+				return;
+			}
+			Ref<Texture2D> tex = _get_grabber_icon();
+			float available_size = vertical ? (get_size().x - tex->get_size().x) : (get_size().y - tex->get_size().y);
+			if (available_size - drag_area_margin_begin - drag_area_margin_end > 0) { // Hides the grabber if it won't fit.
```

Comment:
```suggestion
			if (available_size - drag_area_margin_begin - drag_area_margin_end > 0) { // Draw the grabber only if it fits.
```
More accurate regarding this condition.